lib/cgi/cookie.rb

```diff
@@ -123,6 +132,14 @@ def secure=(val)
       @secure
     end
 
+    # Set whether the Cookie is a httponly cookie or not.
+    #
+    # +val+ must be a boolean.
+    def httponly=(val)
+      @httponly = val if val == true or val == false
```

Comment:
I basically copied what is done for the `secure` flag here. I thought I should be consistent with how the library behaves in similar situations.

Surely I can raise a TypeError if that's what is needed, but then I shall do that for the other flags too.

What do you think?